kolibri/utils/system.py

```diff
@@ -239,27 +237,22 @@ def become_daemon(**kwargs):
     _become_daemon_function = _windows_become_daemon
 
 
-def _symlink_capability_check():
+def symlink_capability_check(source_file, destination_folder):
     """
     Function to try to establish a symlink
+    between two locations
     return True if it succeeds, return False otherwise.
     """
-    # If STATIC_USE_SYMLINKS has been set to False, return False directly
-    if not OPTIONS["Deployment"]["STATIC_USE_SYMLINKS"]:
-        return False
 
-    fd, temp_target = tempfile.mkstemp()
-    temp_pathname = temp_target + ".lnk"
+    temp_pathname = os.path.join(destination_folder, os.path.basename(source_file))
```

Comment:
one thing I notice is that the `temp_pathname` may not be the actual symlink we make in the KOLIBRI_HOME directory.

For example, if `source_file` is `xxx/kolibri/core/static/assets/logo.ico`, then os.path.basename is `logo.ico`, which means the `temp_pathname` would be `~/.kolibri/static/logo.ico`. However, I guess what we want is `~/.kolibri/static/assets/logo.ico`? Or does this not matter?

Reply:
I did this deliberately to avoid making a bunch of child directories in the general case - with the assumption that if we can symlink to the parent directory, then we can symlink to any child directories.

Can't think of any case where that might not be the case.

Reply:
ah sounds good! this makes sense to me!